src/Http.ts

```diff
@@ -137,21 +150,29 @@ function toXHRRequest<A>(req: Request<A>): AjaxRequest {
   }
 }
 
+function parseResponseHeaders(rawRespHeaders: string): Record<string, string> {
```

Comment:
@bmazzarol I think we can add a field to the `Response` object in order to provide access to the original `XMLHttpRequest` object:

```ts
export interface Response<Body> {
  url: string
  status: {
    code: number
    message: string
  }
  headers: Record<string, string>
  body: Body,
  xhr: XMLHttpRequest
}
```

so every user can choose how to handle it (and we do not have to manage extra code in `elm-ts` source)

Reply:
Yep this will solve all my issues, will do.